skyportal/facility_apis/lt.py

```diff
@@ -0,0 +1,799 @@
+import time
+from datetime import datetime, timedelta
+
+from lxml import etree
+from suds import Client
+
+from astropy.coordinates import SkyCoord
+from astropy import units as u
+
+from . import FollowUpAPI
+from baselayer.app.env import load_env
+
+from ..utils import http
+
+env, cfg = load_env()
+
+LT_XML_NS = 'http://www.rtml.org/v3.1a'
+LT_XSI_NS = 'http://www.w3.org/2001/XMLSchema-instance'
+LT_SCHEMA_LOCATION = (
+    'http://www.rtml.org/v3.1a http://telescope.livjm.ac.uk/rtml/RTML-nightly.xsd'
+)
+
+
+class LTRequest:
+
+    """An XML structure for LT requests."""
+
+    def _build_prolog(self):
+        """Payload outline for all LT queue requests.
+
+        Returns
+        ----------
+        payload: etree.Element
+            payload outline for LT requests.
+        """
+
+        namespaces = {
+            'xsi': LT_XSI_NS,
+        }
+        schemaLocation = etree.QName(LT_XSI_NS, 'schemaLocation')
+        uid = format(str(int(time.time())))
```

Comment:
what is this ?

Reply:
it sets the ID for the observation. Maybe the request has something more suitable?

Reply:
`request.id`